src/cli.js

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env node
+//TODO: add a .commentsignore file for files which you would like to ignore from the --list.
```

Comment:
We could creat an issue and remove this comment, I'm not a fan of TODOs, they never go away :)

Reply:
I went on a bit of a "cleaning" spree late at night, and got carried away. I'll recommit it :)